### PR TITLE
[TRAFODION-1542] incorrect port numbers generated for dcs when install_local_hadoop is run using default ports

### PR DESCRIPTION
**Pull request by hegdean**

### CODE REVIEW

File: core/sqf/sql/scripts/install_traf_components [hunk 1]
```diff
@@ -25,17 +25,42 @@ MY_SW_ROOT=$MY_SQROOT/sql/local_hadoop
...
+
+if [ -z "$DEFAULT_PORT_USED" ]; then
+     DEFAULT_PORT_USED="no"
+fi
+
+if [ -z "$MY_START_PORT" ]; then
+    MY_START_PORT=`cat $MY_SQROOT/sql/scripts/sw_env.sh |grep MY_HADOOP_HDFS_PORT_NUM | cut -d "=" -f2`
+fi
+
+if [ $DEFAULT_PORT_USED == 'yes' ]; then
```
Comment (zellerh) [inline core/sqf/sql/scripts/install_traf_components]: This duplicates the code in install_local_hadoop. Would it make sense to add all the ports you need here to sw_env.sh?

**Comment by hegdean** [comment]
Yes. I don't like duplicating the ports either. Should I go ahead and make that change then?

**Comment by hegdean** [comment]
jenkins, retest

**Comment by zellerh** [comment]
Thanks for the changes, this should work very nicely now.

**Comment by selvaganesang** [comment]
Thanks for removing the unwanted messages

**Comment by arvind-narain** [comment]
I'll merge the change in

**Pull request merged**